Form/Type/DocumentType.php

```diff
@@ -20,9 +20,24 @@
 use Doctrine\Bundle\PHPCRBundle\Form\ChoiceList\PhpcrOdmQueryBuilderLoader;
 use Symfony\Bridge\Doctrine\Form\ChoiceList\EntityLoaderInterface;
 use Symfony\Bridge\Doctrine\Form\Type\DoctrineType;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
 
 class DocumentType extends DoctrineType
 {
+    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    {
+        parent::setDefaultOptions($resolver);
+        $type = $this;
+        $loader = function (Options $options) use ($type) {
+            $qb = $options['query_builder'] ?: $options['em']->createQueryBuilder()->fromDocument($options['class'], 'a');
+
+            return $type->getLoader($options['em'], $qb, $options['class']);
```

Comment:
it is now in 2.3.27 - but either we would need to up our minimal requirement to that symfony version (and also somehow avoid 2.6 < 2.6.6) or we could see if we can do a version check at this point and explode if the version is not new enough. or only overwrite if we are on a too old version of symfony. not sure what is best and the most elegant?